README.md

```diff
@@ -118,6 +134,15 @@ container_pull(
 )
 ```
 
+Note: Bazel does not deal well with diamond dependencies. If the repositories that
+are imported by `container_repositories()` have already been imported (at a
+different version) by other rules you called in your `WORKSPACE`, which are
+placed above the call to `container_repositories()`, arbitrary errors might
+ocurr. If you get errors related to external repositories, you will likely
+not be able to use `container_repositories()` and will have to import
+directly in your `WORKSPACE` all the required dependencies (see the most up
+to date impl of `container_repositories()` for details).
```

Comment:
This is happening? container_repositories should check for if the repo has been loaded or not.